extensions/amp-list/0.1/amp-list.js

```diff
@@ -742,14 +719,15 @@ export class AmpList extends AMP.BaseElement {
     return this.fetchList_(/* opt_append */ true)
         .then(() => {
           this.toggleLoadMoreLoading_(false);
-          if (this.unlistenLoadMoreButton_) {
-            this.unlistenLoadMoreButton_();
-            this.unlistenLoadMoreButton_ = null;
+          if (this.unlistenLoadMore_) {
+            this.unlistenLoadMore_();
+            this.unlistenLoadMore_ = null;
           }
         });
   }
 
   /**
+   * @return {!Element|null}
```

Comment:
nit: prefer `{?Element}` over `{!Element|null}` to annotate a [nullable object](https://github.com/google/closure-compiler/wiki/Annotating-JavaScript-for-the-Closure-Compiler#nullable-type).

Reply:
I'll fix this in the next one, upcoming.